### ISSUE TITLE
Support mget in Dyno

### ISSUE THREAD

**Issue by shailesh33** [Netflix/dyno#167]
Support mget. This has following requirements:

-  [ ] Support basic mget

- [ ] Support mget with compression

- [ ] Support scatter gather for token aware mget. This is a little involved because mget will have to split the request into different key sets based on token to query to individual token owners. Today Dynomite takes care of this splitting of request and aggregating of requests

**Comment by balajivenki** [comment]
@shailesh33  and @ipapapa  I see this has been implemented and exist in 1.6.3 itself. However the supported mget is only for String keys. Is there any ETA to support byte[] keys? The BinaryClient has support for mget with byte[]. Please let us know.

**Comment by ipapapa** [comment]
I do not believe we have an internal need right now. Feel free to file a PR if you wish.

**Comment by balajivenki** [comment]
Thanks @ipapapa  I am working on implementing MultiKeyBinaryCommands and basic implementation for mget with byte[]. So far my unit tests are success and seems to be working for both mget with String and byte[]. I will test it further and send you guys a PR.
